src/portal/components/CreateIP.vue

```diff
@@ -185,7 +185,10 @@ export default class CreateIP extends Vue {
     let check_ip = true;
     let check_pub_ip = true;
     this.toIpErrorMessage = "";
-    
+    if (this.toPublicIP === "") {
+      this.ipErrorMessage = "";
```

Comment:
Can we remove this line ? (It returns true so no error will be displayed)
Or we can change the error variable to `toIpErrorMessage` (to make sure it is cleared)

Reply:
changed ipError to toIPError.